src/tests/ftest/erasurecode/ec_offline_rebuild.yaml

```diff
@@ -54,11 +54,11 @@ pool:
     name: daos_server
     scm_size: 500G
     nvme_size: 500G
+    svcn: 1
```

Comment:
What is the default svcn value? Is it not 1?

Reply:
Not now because of changes in ef2bc617720aad27b43ce7d9748dc13a74185931 "The default value for --nsvc is now 3, in order to create resilient pools by default."